modules/module-extras.php

```diff
@@ -82,3 +82,25 @@ function jetpack_widgets_add_suffix( $widget_name ) {
 	);
 }
 add_filter( 'jetpack_widget_name', 'jetpack_widgets_add_suffix' );
+
+add_action( 'blog_privacy_selector', 'priv_notice_privacy_selector' );
+
+/**
+ * Displays the option to enable privacy in Settings.
+ */
+function priv_notice_privacy_selector() {
+	?>
+	<p>
+	<?php
+		printf(
+			'%s <a href="%s#/security?term=private">%s</a>',
+			esc_html__( 'You can also make your site completely private by allowing only registered users to see your site.', 'jetpack' ),
+			esc_url( admin_url( 'admin.php?page=jetpack' ) ),
+			esc_html__( 'Go to Private Sites settings', 'jetpack' )
```

Comment:
Should it use singular?

```suggestion
			esc_html__( 'Go to Private Site settings', 'jetpack' )
```

Reply:
Done in [aaf7862](https://github.com/Automattic/jetpack/pull/10333/commits/aaf78620ceb4629a874763923b9c282fd8c7786f)